### PR TITLE
Even more Graphics articles

### PR DESCRIPTION
**Pull request by SimonDarksideJ**
# Summary

- Added Drawing text

### CODE REVIEW

File: articles/getting_to_know/howto/graphics/HowTo_Draw_Text.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+            Viewport viewport = _graphics.GraphicsDevice.Viewport;
+            
+            // TODO: Load your game content here            
+            fontPos = new Vector2(viewport.Width / 2, viewport.Height / 2);
+            }
+    ```
+
+1. In your [Game.Draw](xref:Microsoft.Xna.Framework.Game#Microsoft_Xna_Framework_Game_Draw_Microsoft_Xna_Framework_GameTime_) method, call [SpriteBatch.Begin](xref:Microsoft.Xna.Framework.Graphics.SpriteBatch#Microsoft_Xna_Framework_Graphics_SpriteBatch_Begin_Microsoft_Xna_Framework_Graphics_SpriteSortMode_Microsoft_Xna_Framework_Graphics_BlendState_Microsoft_Xna_Framework_Graphics_SamplerState_Microsoft_Xna_Framework_Graphics_DepthStencilState_Microsoft_Xna_Framework_Graphics_RasterizerState_Microsoft_Xna_Framework_Graphics_Effect_System_Nullable_Microsoft_Xna_Framework_Matrix__) on the [SpriteBatch](xref:Microsoft.Xna.Framework.Graphics.SpriteBatch) object.
+
+1. If necessary, determine the origin of your text.
```
Comment (kaltinril) [inline articles/getting_to_know/howto/graphics/HowTo_Draw_Text.md:82]: Recommend putting (Center) after origin

"If necessary, determine the origin (center) of your text."

File: articles/getting_to_know/howto/graphics/HowTo_Draw_Text.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
+---
+title: Drawing Text with a Sprite
+description: Demonstrates how to import a SpriteFont into a project and to draw text using DrawString
+requireMSLicense: true
```
Comment (kaltinril) [inline articles/getting_to_know/howto/graphics/HowTo_Draw_Text.md:4]: What does this mean?  Is a Microsoft License required for fonts?  Is there a place that explains this?

Comment (SimonDarksideJ) [reply articles/getting_to_know/howto/graphics/HowTo_Draw_Text.md:4]: :D, this is the front matter for the doc, meaning the original source of the doc is from Microsoft, so we automatically put an MS license note at the end of the article :D

File: articles/getting_to_know/howto/graphics/HowTo_Draw_Text.md (hunk 1 above)
Comment (SimonDarksideJ) [reply articles/getting_to_know/howto/graphics/HowTo_Draw_Text.md:82]: updated, thanks!

**Pull request merged**